src/commands/protocols/common/submit-commit-command.js

```diff
@@ -110,9 +110,26 @@ class SubmitCommitCommand extends Command {
             );
         });
 
+        const sendSubmitCommitTransactionOperationId = this.operationIdService.generateId();
         let txSuccess;
         try {
+            this.operationIdService.emitChangeEvent(
+                OPERATION_ID_STATUS.COMMIT_PROOF.SUBMIT_COMMIT_SEND_TX_START,
+                sendSubmitCommitTransactionOperationId,
+                blockchain,
+                agreementId,
+                epoch,
+                operationId,
+            );
             txSuccess = await transactionCompletePromise;
+            this.operationIdService.emitChangeEvent(
```

Comment:
Do we want to emit END event if tx fails because commit was already submitted or maybe we should use some other status?

Reply:
If transaction fails we will emit error event

Reply:
But we're not going into catch clause for errors that are "Already submitted"